main.go

```diff
@@ -172,6 +173,16 @@ func _main(ctx context.Context) error {
 			DBName:       cfg.PGDBName,
 			QueryTimeout: cfg.PGQueryTimeout,
 		}
+		// Replace {netname} with curnet
+		if strings.Contains(dbi.DBName, "{netname}") {
+		  reg, err := regexp.Compile("[^a-zA-Z]+")
```

Comment:
What is the reasoning for removing other valid DB name characters like numerals?

Reply:
I thought that if TestNet3 gets bumped to TestNet4 that it should use the same DB.  That way we can test any reorg that would get run on MainNet instead of renaming the DB.  This might not matter since most of the people that run this code are Devs and can do that pretty easily.

Reply:
The thing with testnet3 -> testnet4 is that they are entirely different blockchains with different genesis blocks.  Starting up on testnet4 for the first time using an existing testnet3 database won't work.